pkgs/applications/office/fava/default.nix

```diff
@@ -0,0 +1,73 @@
+{ stdenv, pkgs, fetchurl, python3Packages }:
+
+let
+
+  # We're building beancount here as dependency-only package because fava 0.3.0
+  # depends on beancount-2.0b2, but beancount as in pkgs.beancount is from the
+  # master branch of the beancount repository (as the maintainer does not
+  # release versions).
+  beancount = python3Packages.buildPythonPackage rec {
```

Comment:
In this case an application, IMO, as it is required as dependency of `fava` - not completely sure, though.